js/customer.js

```diff
@@ -5,91 +5,69 @@ window.addEventListener('load', function () {
-        /*==============================================================*/
-        if (validName(name) && validPhoneNumber(phoneNumber) && validEmail(email)) {
-                    alert('Thank you for register with us');
-        // If the name, email and contact number are not valid execute this code
-        /*====================================================================*/
-        }else if (!validName(name) && !validPhoneNumber(phoneNumber) && !validEmail(email)) {
-                    name.style.background = '#f00';
-                    email.style.background = '#f00';
-                    phoneNumber.style.background = '#f00';
-        // If the name and email are valid but phone number not execute this code
-        /*====================================================================*/
-        }else if(validName(name) && !validPhoneNumber(phoneNumber) && validEmail(email)) {
-                    phoneNumber.style.background = '#f00';
-        // If the name and phone number are valid but email not execute this code
-        /*==================================================================*/
-        }else if(validName(name) && validPhoneNumber(phoneNumber) && !validEmail(email)) {           
-                    email.style.background = '#f00';
-        // If the email and phone number are valid but name not execute this code
-        /*==================================================================*/
-        }else  if(!validName(name) && validPhoneNumber(phoneNumber) && validEmail(email)) {
-                    name.style.background = '#f00';
-        // If the name is valid but email and phone number are not execute this code
-        /*==================================================================*/
-        }else if(validName(name) && !validPhoneNumber(phoneNumber) && !validEmail(email)) {
-                    email.style.background = '#f00';
-                    phoneNumber.style.background = '#f00';
-        // If the email is valid but name and phone number are not execute this code
-        /*==================================================================*/
-        } else if(!validName(name) && !validPhoneNumber(phoneNumber) && validEmail(email)) {
-                    name.style.background = '#f00';
-                    phoneNumber.style.background = '#f00';
-        // If the phone number is valid but email and name are not execute this code
-        /*==================================================================*/
-        }else if(!validName(name) && validPhoneNumber(phoneNumber) && !validEmail(email)) {
-                    name.style.background = '#f00';
-                    email.style.background = '#f00';
-        }else {
-                    alert('error');
-                    return false;
+        if (validName(name) && validName(name) && validPhoneNumber(phoneNumber)) { alert('Thank you for register with us')}
+
+        if (validName(name)) {
+                return true;
```

Comment:
this is wrong logic .. if you have `return` then you will never get to the next line - you will never remove the calss 'error'. There is no need to `return` in any of these checks.